ui/src/app/settings/containers/Network/NetworkDiscoveryForm/NetworkDiscoveryForm.test.js

```diff
@@ -17,7 +17,7 @@ describe("NetworkDiscoveryForm", () => {
         items: [
           {
             name: "active_discovery_interval",
-            value: 0,
+            value: "0",
```

Comment:
This was an unrelated props bug, not sure how this landed? 😕

Reply:
how does this bug manifest itself?

Reply:
@steverydz a warning in the console from proptypes